### ISSUE TITLE
Check the info button content for typos.

### ISSUE THREAD

**Issue by PatReynolds** [FreeUKGen/FreeBMD2#248]

**Comment by DeniseColbert** [comment]
@ComfE-2662 is going to take a look at this

**Comment by ComfE-2662** [comment]
I was unable to complete this task because:

1. I am unable to sign in as a member on the test 3 site
2. The site only seems to be showing basic html with no css or styling to it
3. I wasn't 100% sure where to find the page - on test2 or test3 and I wasn't sure the exact page tbh

I think I'll need a bit of guidance at the next meeting next week

**Comment by ComfE-2662** [comment]
I have made a folder in the shared drive called 'New work' so that anything I do can be take out of there and put in the right place and also to be found easily.
I checked the search page and created a spreadsheet of my findings. I haven't edited them as I am unable to due to access but at least the person who does edit them can just copy and paste my suggested edits. 
I think the main issue was that the font size was just too big and the content too wordy

**Comment by DeniseColbert** [comment]
Typos seem to be fixed, thanks @Vino-S. 

Comfort has highlighted changes to text, can be seen here: https://drive.google.com/file/d/1dVL2r9tfVV1bjf2iS8y6_SULz4pk-WVu/view?usp=sharing

Use of asterisk to denote non-optional fields needs discussion

**Comment by richpomfret** [comment]
@PatReynolds to take a look and see if any next steps are needed or if we can close this as completed.

**Comment by PatReynolds** [comment]
Hi @ComfE-2662 thanks for this - very helpful.

All fields are optional, so we don't need to mark them as optional/required in any way.  What we should do is say at the top 'all fields are optional'.  

If we say at the top 'you can use upper or lower case letters anywhere' then it doesn't have to go in any infobubble. 

I don't think 'date range from' and 'date range to' need an info button.  Or possibly to refer elsewhere to the months/quarters useage by the GRO? As per story #185

Label for age at death will need some work after Issue #225 is complete.

**Comment by PatReynolds** [comment]
Hi @Vino-S please could you implement the changes as susggested by comfort here: https://drive.google.com/file/d/1dVL2r9tfVV1bjf2iS8y6_SULz4pk-WVu/view?usp=sharing _except_ those referring to 'optional' .  please remove all mentions of 'optional'.

**Comment by Vino-S** [comment]
In progress

**Comment by Vino-S** [comment]
Ready for testing in test2

**Comment by PatReynolds** [comment]
There is  no blank in the quarter fields for date range.

Spouse's surname doesn't seem to work (no additional info shows).

There's a missing space on the Age at Death but I'm wondering if we need to have any additional help here at all - the dropdown shows the options, which seem self-explanatory.  Let's discuss in review.

Otherwise all good, thank you, @Vino-S

**Comment by KristinaGadzhieva** [comment]
Can be closed, there are no typos.
